### PR TITLE
Us 16

### PR DESCRIPTION
**Pull request by nmking22**
- [x] done

User Story 16, User can log out

As a registered user, merchant, or admin
When I visit the logout path
I am redirected to the welcome / home page of the site
And I see a flash message that indicates I am logged out
Any items I had in my shopping cart are deleted

### CODE REVIEW

File: spec/features/sessions/delete_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                        address: "Some dark cave 11",
+                        city: "Arkham",
+                        state: "CO",
+                        zip: "81301",
+                        email: 'batmansemail@email.com',
+                        password: "password")
+    # bike_shop = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+    # tire = bike_shop.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+    #
+    # user.items << tire
```
Comment (helloeduardo) [inline spec/features/sessions/delete_spec.rb:15]: Thanks for adding these comments, I made issue #18 to address it.

**Review by helloeduardo — APPROVED** [review]
Logging out looks great! 💻🙅‍♂️

**Review by the-color-bliu — APPROVED** [review]
Nice job here, good of use of prying your way through.

**Review by BJSherman80 — COMMENTED** [review]
great job code looks clean

**Pull request merged**